pom.xml

```diff
@@ -80,9 +80,12 @@
     <commons.version>3.9</commons.version>
     <common.codec.version>1.12</common.codec.version>
     <gson.version>2.8.5</gson.version>
+    <googleauth.version>1.6.0</googleauth.version>
+    <guava.version>31.1-jre</guava.version>
```

Comment:
I'd try to use guava version from hydrator. Otherwise strange things may happen

Reply:
And BTW, where are you using it? I don't see guava references in the source

Reply:
The GoogleCredentials is asking for one of the package from guava. It won't let me build before I include it.

Reply:
Hm. Why don't they include it themselves? Usually it's up to an artifact to include any dependency needed